xmtp_mls/src/identity_updates.rs

```diff
@@ -117,10 +117,18 @@ where
             return Err(AssociationError::MissingIdentityUpdate.into());
         }
 
-        if let Some(association_state) =
-            StoredAssociationState::read_from_cache(conn, inbox_id.to_string(), last_sequence_id)?
+        match StoredAssociationState::read_from_cache(conn, inbox_id.to_string(), last_sequence_id)
         {
-            return Ok(association_state);
+            Ok(Some(state)) => {
+                return Ok(state);
+            }
+            // If the cache fails to deserialize, it's because the structure of the cache
+            // has changed, and we need to reload.
```

Comment:
I'm not 100% sure that deserialization will fail if the `AssociationStateProto` (via the `MemberProto`) has been [updated](https://github.com/xmtp/proto/pull/208/files) to include a new optional field. So I think we still need a way to skip the cache after this update.

Reply:
The bytes from the sql blob won't align with the proto. It should fail.

Reply:
That sounds right. Protobuf is very forgiving

Reply:
Created a migration to clear out the table in question.